lib/cli/src/Cardano/CLI.hs

```diff
@@ -177,11 +179,8 @@ import Cardano.Wallet.Primitive.AddressDerivation
     , PassphraseMaxLength
     , PassphraseMinLength
     , WalletKey (..)
-    , XPrv
-    , XPub
     , deriveRewardAccount
     , hex
-    , unXPrv
     , unXPrvStripPubCheckRoundtrip
     , xPrvFromStrippedPubXPrvCheckRoundtrip
```

Comment:
If `xprvFromBytes` never modifies keys (but fails), we could replace `xPrvFromStrippedPubXPrvCheckRoundtrip` and `unXPrvStripPubCheckRoundtrip` too I suspect.

Reply:
done in https://github.com/input-output-hk/cardano-wallet/pull/1640/commits/e9d860edc854e575712c720405e06aa515ea77f7

Reply:
a6ba786d7bdd1a3128161789ba4ed6a4380dceea Ok, so I guess it didn't work.

Btw, have we thought about whether `xprvFromBytes` is dangerous? Can users of `cardano-addresses` accidentally mess up by having their keys silently modified ("tweaked")?

Reply:
to my best knowledge, as the `cardano-address` implementation works with Byron key, which wasn't the case for `cardano-wallet` impl, we are fine here with this change.

Reply:
but i need to change the expected behavior of two  Keys tests in CLI as they exploited the tweak present in `cardano-wallet` - see my comment (asking for double-check) below for those tests

Reply:
> to my best knowledge, as the cardano-address implementation works with Byron key, which wasn't the case for cardano-wallet impl, we are fine here with this change.

Ok, good, that was what I hoped.

I tried this to be completely convinced, and it passes

```haskell
prop_roundtrip
    :: XPrv
    -> Index 'WholeDomain 'AddressK
    -> Property
prop_roundtrip xprv ix =
    xprvFromBytes (xprvToBytes k) === Just k
  where
    rootXPrv = genMasterKeyFromXPrv xprv :: Byron 'RootK XPrv
    k = deriveXPrv DerivationScheme1 (getKey rootXPrv) ix
```